core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/datalake/upgrade/validation/config/ClusterUpgradeDiskSpaceValidationEventToClusterUpgradeImageValidationFinishedEventConverter.java

```diff
@@ -0,0 +1,19 @@
+package com.sequenceiq.cloudbreak.core.flow2.cluster.datalake.upgrade.validation.config;
+
+import com.sequenceiq.cloudbreak.core.flow2.cluster.datalake.upgrade.validation.event.ClusterUpgradeDiskSpaceValidationEvent;
+import com.sequenceiq.cloudbreak.core.flow2.cluster.datalake.upgrade.validation.event.ClusterUpgradeImageValidationFinishedEvent;
+import com.sequenceiq.flow.core.PayloadConverter;
+
+public class ClusterUpgradeDiskSpaceValidationEventToClusterUpgradeImageValidationFinishedEventConverter
+        implements PayloadConverter<ClusterUpgradeImageValidationFinishedEvent> {
+    @Override
+    public boolean canConvert(Class<?> sourceClass) {
+        return ClusterUpgradeDiskSpaceValidationEvent.class.isAssignableFrom(sourceClass);
+    }
+
+    @Override
+    public ClusterUpgradeImageValidationFinishedEvent convert(Object payload) {
+        ClusterUpgradeDiskSpaceValidationEvent sourcePayload = (ClusterUpgradeDiskSpaceValidationEvent) payload;
+        return new ClusterUpgradeImageValidationFinishedEvent(sourcePayload.getResourceId(), 1L);
```

Comment:
this would mean that the validation would pass for sure. is it the desired behavior?
I couldn't find anything about this in the commit message. maybe a bit more descriptive message would be useful in general